integration-tests/container-image/pom.xml

```diff
@@ -9,6 +9,9 @@
   </parent>
   <artifactId>quarkus-integration-test-container-image</artifactId>
   <name>Quarkus - Integration Tests - Jib extension</name>
```

Comment:
The name looks a bit off?

Reply:
Indeed, thanks for spotting it! It should be fixed now